.github/workflows/build-dev.yaml

```diff
@@ -81,6 +81,7 @@ jobs:
           - "clusterctl-upgrade"
           - "clusterclass"
           - "capx-feature-test"
+          - "image-templating"
```

Comment:
We are already running this test as part of `capx-feature-test` suite. We shouldn't have to run this twice in CI.

Reply:
resolved